Backend/src/server.test.ts

```diff
@@ -0,0 +1,11 @@
+import supertest from 'supertest';
+import { server } from './server';
+
+const request = supertest(server);
+
+it('Gets / endpoint', async () => {
```

Comment:
Message should be like this:
`it('should <description what should happen>)`

If a test fails the message starts with "it" so it makes more sense to write descriptions like this